tests/market/test_market.py

```diff
@@ -335,26 +335,6 @@ def test_market_acct_multiple(market=OneSidedMarket(bc=NonBlockchainInterface(st
     assert market.bought_energy("C") == offer2.energy == 10
 
 
-@pytest.mark.parametrize("market, offer", [
-    (OneSidedMarket(bc=NonBlockchainInterface(str(uuid4())), time_slot=now()), "offer"),
-    (BalancingMarket(bc=NonBlockchainInterface(str(uuid4())), time_slot=now()), "balancing_offer"),
-    (SettlementMarket(bc=NonBlockchainInterface(str(uuid4())), time_slot=now()), "offer"),
-])
-def test_market_avg_offer_price(market, offer):
-    getattr(market, offer)(1, 1, "A", "A")
-    getattr(market, offer)(3, 1, "A", "A")
-
-    assert market.avg_offer_price == 2
-
-
-@pytest.mark.parametrize("market",
-                         [OneSidedMarket(bc=MagicMock(), time_slot=now()),
-                          BalancingMarket(bc=MagicMock(), time_slot=now()),
-                          SettlementMarket(bc=MagicMock(), time_slot=now())])
-def test_market_avg_offer_price_empty(market):
-    assert market.avg_offer_price == 0
```

Comment:
Since these tests were deleted, and by a brief look at the code, I saw that there are some unused members in the market class that we could get rid of. These are `_avg_offer_price`, `min_offer_price`, `max_offer_price`. Also `_update_min_max_avg_offer_prices` method should be deleted as far as I see. Please double check in order to validate that I did not miss any use of these members outside of the class, I did not see any such occurrences from a first look though.

Reply:
Thanks for looking them up! you're right!
I pushed their deprecation